e2e/Microsoft.Azure.Devices.E2ETests/X509E2ETests.NetFramework.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Azure.Devices.Client;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.E2ETests
+{
+    public partial class X509E2ETests
+    {
+        private async Task SendSingleMessageX509(Client.TransportType transport)
```

Comment:
Same as above: please move the PAL API up and delimit it from the platform-specific implementation.